doc/add-data-new-version.md

```diff
@@ -4,46 +4,42 @@ For bedrock edition see [bedrock.md](bedrock.md)
-| blockCollisionShapes.json | Yes | Use [McDataExtracting][4] |
-| commands.json | No? |Use [mc-data-command-generator][5] | link to jar files have to be manually added |
-| biomes.json | Yes | Use [Burger][1], then use [burger-extractor][2] | [extra info][13] |
-| instruments | No | [wiki.vg block actions][11] |
-| materials.json | No | [import from previous version](18) or [fix it properly](19) |
-| windows.json | No | Look at [wiki.vg inventories][7] |
-| version.json | No | Look at [Protocol Versions][9] | [wiki.vg protocol numbers][8] |
-| effects.json | No | Look at [minecraft wiki effects][12] |
-| enchantments.json | No | Duplicate enchantments.json from the latest version and add manually the missing enchantments. Enchantments data could be found by looking into the deobfuscated classe files of each enchantments, as well as in the Enchantments registry. |
-| language.json | Yes | Use [minecraft-jar-extractor][10] |
-| particles.json | Yes | Use [Burger][1], then use [burger-extractor][2] |
-| blockLoot.json | Yes | Use [minecraft-jar-extractor][10] |
-| entityLoot.json | Yes | Use [minecraft-jar-extractor][10] |
-| mapIcons.json | No | Icons can be found in the Minecraft jar file where they are added as a single sprite. The file location is `/assets/minecraft/textures/map/map_icons.png`. Alternatively you might be able to look up the icons from the following page on the [Minecraft wiki][15] or from [wiki.vg][16]. | [minecraft-data pr mapIcons][14] |
+| protocol.json | No | Follow [Pre-release Protocol][1] | Always double check by looking at the Minecraft source files. |
+| blocks.json | Yes | Use [minecraft-data-generator-server][2] |
+| items.json | Yes | Use [minecraft-data-generator-server][2] |
+| entities.json | Yes | Use [minecraft-data-generator-server][2]  and run `extractPcEntityMetadata.js` script in tools/js to generate entity metadata in entities.json and protocol.json |
+| recipes.json | Yes | Use [minecraft-data-generator-server][2] | Make sure that `-1` in output are replaced with `null` |
+| blockCollisionShapes.json | Yes | Use [minecraft-data-generator-server][2] |
+| commands.json | No? |Use [mc-data-command-generator][3] | Link to jar files have to be manually added |
+| biomes.json | Yes | Use [minecraft-data-generator-server][2] |
+| instruments | Yes | Use [minecraft-data-generator-server][2] |
+| materials.json | Yes | Use [minecraft-data-generator-server][2] |
+| windows.json | No | Look at [wiki.vg inventories][4] |
+| version.json | No | Look at [Protocol Versions][5] | [wiki.vg protocol numbers][6] |
+| effects.json | Yes | Use [minecraft-data-generator-server][2] |
+| enchantments.json | Yes  | Use [minecraft-data-generator-server][2] |
+| language.json | Yes | Use [minecraft-data-generator-server][2] |
+| particles.json | Yes | Use [minecraft-data-generator-server][2] |
+| blockLoot.json | No | Currently no extractor seems to works for this |
```

Comment:
minecraft-jar-extractor should be mentioned even if it needs updates ; until we have something better

Reply:
Problem is minecraft-jar-extractor doesn't work at all for anything on current versions of MC.  That's why we have been using blockLoot.json and entityLoot.json from 1.17 on every version since. (At least that is my guess why.)

I only removed it since you said to not list it while also saying no extractor currently works.

Reply:
Okay, just did some testing, seems the extractor will work for 1.20.3 but not 1.20.4, 1.20.2 or 1.20 it self which makes no sense at all (Throwing timeouts errors on getting the jars or data from mojang). So I will update this to say to use it but that it may have require fixing per versions.